integration-tests/emulated/tests/governance/polkadot/src/open_gov_on_asset_hub.rs

```diff
@@ -299,3 +299,63 @@ fn assethub_can_authorize_upgrade_for_system_chains() {
 		)
 	});
 }
+
+#[test]
+fn assethub_fellowsip_admin_can_induct_on_collectives() {
+	type AssetHubRuntime = <AssetHubPolkadot as Chain>::Runtime;
+	type AssetHubCall = <AssetHubPolkadot as Chain>::RuntimeCall;
+	type AssetHubOrigin = <AssetHubPolkadot as Chain>::RuntimeOrigin;
+
+	type CollectivesRuntime = <CollectivesPolkadot as Chain>::Runtime;
+	type CollectivesRuntimeCall = <CollectivesPolkadot as Chain>::RuntimeCall;
+	type CollectivesRuntimeEvent = <CollectivesPolkadot as Chain>::RuntimeEvent;
+
+	use asset_hub_polkadot_runtime::governance::pallet_custom_origins::Origin::FellowshipAdmin;
+	use sp_keyring::Sr25519Keyring::Charlie;
+
+	let account_to_induct: <CollectivesRuntime as frame_system::Config>::AccountId =
+		Charlie.to_account_id().into();
+
+	let send_induct_over_xcm = build_xcm_send_induct_member::<
+		AssetHubPolkadot,
+		CollectivesPolkadot,
+		pallet_core_fellowship::Instance1,
+	>(
+		AssetHubPolkadot::sibling_location_of(<CollectivesPolkadot as Parachain>::para_id()),
+		account_to_induct.clone(),
+		None,
+	);
+
+	let fellows_origin: AssetHubOrigin = FellowshipAdmin.into();
```

Comment:
either `fellowship_admin` or `fellowship_admin_origin`, because `fellows_origin` is used already in the code `Fellows` as Rank3+ member, so let's avoid any confusion:
```
        /// Origin commanded by Polkadot Fellows (3rd Dan fellows or greater).
		Fellows,
```


```suggestion
	let fellowship_admin: AssetHubOrigin = FellowshipAdmin.into();
```

Reply:
This was some work in progress and I've actually wanted to also introduce "bad" scenarios - so I have changed this since then and its now `ok_origin` and `bad_origin`